cast/src/lib.rs

```diff
@@ -1050,6 +1054,24 @@ impl SimpleCast {
         Ok(format!("0x{}{}", "0".repeat(64 - num_hex.len()), num_hex))
     }
 
+    pub fn left_shift(value: &str, bits: &str, base_in: u32) -> Result<U256> {
+        let value = U256::from_str_radix(value, base_in)
+            .wrap_err("Cannot parse input with provided base_in")?;
+        let bits = U256::from_str_radix(bits, base_in)
```

Comment:
Do we want the shift to be hex/dec too? Afaik you mostly just do shifts in dec

Reply:
Yes, that's stupid you're right !